en/drafts/originals/creating-guis-in-python-for-digital-humanities-projects.md

```diff
@@ -127,151 +93,102 @@ _Linux Users_: Use the following pip commands to install Qt Designer:
+If you cannot locate the application, follow the instructions above in the [Prerequisites and Tools section](#prerequisites-and-tools) to download and install Qt Designer.
 
-After the application loads, a prompt will request you to create a New Form.
-To simplify this project, we will select “Dialog without Buttons.” Ensure that “Show this Dialog on Startup” is selected, and then press Create.
+After the application loads, a prompt will request you to create a New Form. To simplify this project, we will select “Dialog without Buttons.” Ensure that “Show this Dialog on Startup” is selected, and then press Create.
 
-The Windows and Linux version of Qt Designer (Figure 2) is self-contained and provides little flexibility using the interface, but it is neatly organized out of the box.
-The MacOS version (Figure 3) has many individual windows and the arrangement is initially cluttered. This tutorial primarily uses the MacOS version
-and it is recommended that you arrange the individual windows in the layout shown in Figure 2 to ease visibility and usage. Whichever version you use, however,
-there is little to no difference relating to following the tutorial. At most, the arrangement of certain items or buttons may differ.
+The Windows and Linux version of Qt Designer (Figure 2) is self-contained and provides little flexibility using the interface, but it is neatly organized 'out of the box'. The MacOS version (Figure 3) has many individual windows and the arrangement is initially cluttered. This tutorial primarily uses the MacOS version and it is recommended that you arrange the individual windows in the layout shown in Figure 2 to ease visibility and usage. Whichever version you use, however, there is little to no difference when it comes to following this tutorial. At most, the arrangement of certain items or buttons may differ.
 
 
-{% include figure.html filename="https://github.com/programminghistorian/ph-submissions/blob/gh-pages/images/creating-guis-in-python-for-digital-humanities-projects/CREATING-GUIS-IN-PYTHON-FOR-DIGITAL-HUMANITIES-PROJECTS2.png?raw=true" caption="Figure 2. The Qt Designer Interface (Windows and Linux Version)" %}
+{% include figure.html filename="CREATING-GUIS-IN-PYTHON-FOR-DIGITAL-HUMANITIES-PROJECTS2.png" alt="Visual description of figure image" caption="Figure 2. The Qt Designer Interface (Windows and Linux Version)" %}
 
-{% include figure.html filename="https://github.com/programminghistorian/ph-submissions/blob/gh-pages/images/creating-guis-in-python-for-digital-humanities-projects/CREATING-GUIS-IN-PYTHON-FOR-DIGITAL-HUMANITIES-PROJECTS3.png?raw=true" caption="Figure 3. The Qt Designer Interface (MacOS Version)" %}
+{% include figure.html filename="CREATING-GUIS-IN-PYTHON-FOR-DIGITAL-HUMANITIES-PROJECTS3.png" alt="Visual description of figure image" caption="Figure 3. The Qt Designer Interface (MacOS Version)" %}
```

Comment:
@DerDoktorFaust Alt-text here too.

Reply:
alt="Qt Designer application containing four numbered rectangular windows. One: GUI interface editor. Two: Widget Box. Three: Property Editor. Four: Signal and Slot Editor."